### PR TITLE
Add Elk-M1 sensor platform documentation

### PR DESCRIPTION
**Pull request by gwww**
**Description:**
See component PR for full details: https://github.com/home-assistant/home-assistant/pull/16952

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#17342
## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
Thanks, @gwww! 👍 

✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Review by frenck — DISMISSED** [review]

File: source/_components/sensor.elkm1.markdown [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+layout: page
+title: "Elk-M1 Sensor"
+description: "Instructions on how to integrate Elk-M1 counters, keypads, panel, settings, and zones."
+date: 2018-10-11 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: elkproducts.png
+ha_release: 0.82
```
Comment (MartinHjelmare) [inline source/_components/sensor.elkm1.markdown]: 0.81

Comment (gwww) [reply source/_components/sensor.elkm1.markdown]: OK. I'm curious now. How would I ever be able to get the version right?

Comment (frenck) [reply source/_components/sensor.elkm1.markdown]: @gwww You won't... not until the parent is merged.
A good start is to pick the "next" major version on creation.

**Review by frenck — APPROVED** [review]

**Pull request merged**